src/override/functions.js

```diff
@@ -0,0 +1,63 @@
+export const getResource = (resources, resourceName) =>
+  resources.find(({name}) => resourceName === name);
+
+export const getResourceField = (resource, resourceFieldName) =>
+  resource.fields.find(({name}) => resourceFieldName === name);
+
+export const overrideInput = (resource, resourceFieldName, overrideInput) => {
+  const resourceField = getResourceField(resource, resourceFieldName);
+
+  if (resourceField === undefined) {
```

Comment:
```suggestion
  if (undefined === resourceField) return;
```

We use Yoda style, and we inline `if` if there is only one line in the block.

Same everywhere